src/invalid.ts

```diff
@@ -0,0 +1,79 @@
+import {SignalRef} from 'vega';
+import {ScaleChannel} from './channel';
+import {Mark, MarkDef} from './mark';
+import {isObject} from 'vega-util';
+
+/**
+ * Mixins for Vega-Lite Spec's Mark Definiton (to add mark.invalid)
+ */
+export interface MarkInvalidMixins {
+  /**
+   * Invalid data mode for marks, which defines how the visualization should represent invalid values (`null` and `NaN` in continuous scales without defined output for invalid values) in the marks and their scale domains.
+   *
+   * -  `"filter"` — *Exclude* all invalid values from the visualization's *marks* and *scales*.
```

Comment:
If I want these:
* ignore domain
* connect nulls with dotted lines.
How can we do it with these modes? I wonder if we should have "ignore-domain-connect-nulls" (temp name) mode and allow the users to customize how to connect the nulls, dotted, dashed, solid lines.

Reply:
>connect nulls with dotted lines.

As discussed earlier, this is super complicated to implement so it's out of scope.